kai/reactive_codeplanner/agent/analyzer_fix/agent.py

```diff
@@ -71,7 +71,6 @@ class AnalyzerAgent(Agent):
 {% for incident in incidents %}
 ### incident {{ loop.index0 }}
 incident to fix: "{{ incident.message | safe }}"
-Line number: {{ incident.line_number }}
```

Comment:
I assume this is because the line may have changed in the actual file, causing some grief/issues. 

I wonder if the task manager could "replace" the similar issues when the line numbers are different, and we could keep this?

Reply:
reverted this change